### ISSUE TITLE
Preview your published site link

### ISSUE THREAD

**Issue by AndreaMonzini** [silexlabs/Silex#652]
Hello,
testing `https://editor.silex.me/` publishing with FTP

wen i publish the website and when i click on "Preview your published site here" the link is:

https://editor.silex.me/api/1.0/ftp/exec/get///silex/Pub/index.html

It's very slow and not pointing to the actual website published link.

![screen shot 2017-11-03 at 6 35 52 pm](https://user-images.githubusercontent.com/12413409/32388009-97148496-c0c6-11e7-9292-c254d7465879.png)

**Comment by lexoyo** [comment]
> It's very slow and not pointing to the actual website published link.

Yes, it is your FTP via Silex server... This is because we do not know the final URL of your site :-/

**Comment by AndreaMonzini** [comment]
The FTP access and the FTP publishing path are not enough to know the published website address?
Maybe could be useful to add manually somewhere the final website address.
Can i use one FTP to load assets and another FTP to publish website?

**Comment by singchan** [comment]
> The FTP access and the FTP publishing path are not enough to know the published website address?

For now silex knows how to read and write with the ftp protocol, e.g. login:pass@ftp.host.com and the publication path you selected, e.g. /vhosts/site1/htdocs/publish1/

But it would require the domain name and the root path of the vhost and the protocol in order to build https://mysite.com/publish1/

> Maybe could be useful to add manually somewhere the final website address.

Yes that is an idea... One could add the domain, the protocol and the root path and if provided we use it

> Can i use one FTP to load assets and another FTP to publish website?

When you publish, silex gather all assets and put them in assets, so currently no

**Comment by JbIPS** [comment]
>  Can i use one FTP to load assets and another FTP to publish website?

Actually, with the new version of CloudExplorer (the file explorer in Silex) you could theoretically open 2 different FTP sessions so that could be possible. We'll have to try it out 😉
